packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -216,10 +215,13 @@ class DependencyExtractionWebpackPlugin {
 			const assetData = {
 				// Get a sorted array so we can produce a stable, stringified representation.
 				dependencies: Array.from( entrypointExternalizedWpDeps ).sort(),
-				version: hash.digest( 'hex' ).substring( 0, 32 ),
+				version,
 			};
 
 			const assetString = this.stringify( assetData );
+			const contentHash = createHash( 'sha512' )
+				.update( assetString )
+				.digest( 'hex' );
```

Comment:
I would be curious to learn whether we could also refactor this usage to let webpack control it:
```suggestion
			const contentHash = createHash( compilation.outputOptions.hashFunction )
				.update( assetString )
				.digest( compilation.outputOptions.hashDigest );
```

Reply:
Yes, I pushed a commit that does that with the `.asset.php` file content hash, too :+1: